### PR TITLE
Less Framework, styling.

### PR DESCRIPTION
**Pull request by JulianNorton**
Bootstrap Less Framework, homepage framework, css exportations, google web-font

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,110 +3,114 @@
...
 <!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-        <title>Buttercoin</title>
-        <meta name="description" content="">
-        <meta name="viewport" content="width=device-width">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
```
Comment (PartTimeLegend) [inline public/index.html]: This is a HTTP Header and not a meta tag. Until w3c says it's valid I'll always reject it.

File: public/index.html [hunk 2]
```diff
@@ -3,110 +3,114 @@
...
-                      </td>
-                    </tr>
-                  </table>
-                </form>
-            <hr>
-            <footer>
-                <p>&copy; Buttercoin 2013</p>
-            </footer>
+      <footer>
+        <p>© Buttercoin 2013</p>
```
Comment (PartTimeLegend) [inline public/index.html]: Use `&copy;`

File: public/index.html [hunk 3]
```diff
@@ -3,110 +3,114 @@
...
-                            </li>
-                        </ul>
-                    </div><!--/.nav-collapse -->
-                </div>
-            </div>
+      <!-- Example row of columns -->
+      <div class="row">
+        <div class="span4">
+          <h2>Heading</h2>
+          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
```
Comment (PartTimeLegend) [inline public/index.html]: I hated Latin at school. Even more so now.

File: public/index.html (hunk 2 above)
Comment (JulianNorton) [inline public/index.html]: why? my understanding is that if it's unnecessary to use the character code, then don't, and it's a character many fonts support.

**Pull request closed without merging**